src/main/java/dao/impl/OrderDaoImpl.java

```diff
@@ -1,18 +1,66 @@
 package dao.impl;
 
 import dao.OrderDao;
-import database.Database;
+import model.Code;
 import model.Order;
+import model.User;
 import org.apache.log4j.Logger;
+import util.DBConnector;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
 
 public class OrderDaoImpl implements OrderDao {
 
     private static final Logger logger = Logger.getLogger(OrderDaoImpl.class);
+    private static final String ADD_ORDER_IN_DB = "INSERT INTO `order` (first_name, last_name, number_phone, street_name, " +
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.4-column-limit
in all classes

Reply:
My column limit is 120, yes. I added the changing.